src/api/.keystone/config.js

```diff
@@ -261,6 +319,39 @@ var extendGraphqlSchema = (schema) => (0, import_schema.mergeSchemas)({
           return { success: false };
         }
       }
+    },
+    Query: {
+      getCompaniesHouseInformation: async (root, variables) => {
+        try {
+          const { companiesHouseNumber } = variables;
+          console.info("Calling Companies House API for ", companiesHouseNumber);
+          const sanitisedRegNo = companiesHouseNumber.padStart(8, "0");
```

Comment:
Might be a good idea to convert to String for input sanatisation.
```
// Converting CHN to String datatype for padding
companiesHouseNumber.toString().padStart(8, "0");

Reply:
done - will always be a string regardless